src/components/CircleProgressBar/index.js

```diff
@@ -17,19 +17,32 @@ const CircleProgressBar = React.createClass({
 
   componentDidMount () {
     this.initProgressBar();
-    this.updateProgressBar(this.props.percentage);
+    if (this.props.percentage > 0) {
+      this.updateProgressBar(this.props.percentage);
+    }
+  },
+
+  componentDidUpdate (prevProps) {
+    if (prevProps.percentage !== this.props.percentage) {
+      this.updateProgressBar(this.props.percentage);
+    }
```

Comment:
ping @artibella - update the progress bar when the component updates (i.e when we fetch a new petition from the state)